packages/aws-cdk-lib/pipelines/test/compliance/synths.test.ts

```diff
@@ -186,6 +186,27 @@ test('CodeBuild: environment variables specified in multiple places are correctl
     }),
   });
 
+  new ModernTestGitHubNpmPipeline(pipelineStack, 'Cdk-2', {
```

Comment:
This existing pipelines test had to be changed to not throw an error - all I did was move the stack mutations before the `Template.fromStack()` calls

Reply:
That's perfect!